src/components/tooltips/line-popover.js

```diff
@@ -0,0 +1,290 @@
+                                                              )
+                                                          )
+                                                        : checkValue(
+                                                              Math.round(
+                                                                  (lineInfo.i2 *
+                                                                      100) /
+                                                                      temporaryLimit.value
+                                                              )
+                                                          ))}
+                                            </TableCell>
+                                            <TableCell>
+                                                {checkValue(
+                                                    side === '1'
+                                                        ? lineInfo.voltageLevelId1
+                                                        : lineInfo.voltageLevelId2
+                                                )}
+                                            </TableCell>
+                                        </TableRow>
+                                    )
+                            )}
+                    </>
+                )}
+            </>
+        );
+    };
+
+    return (
+        <Popover
+            anchorEl={anchorEl}
+            sx={{
+                pointerEvents: 'none',
+            }}
+            anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'left',
+            }}
+            transformOrigin={{
+                vertical: 'top',
+                horizontal: 'right',
+            }}
+            onClose={handlePopoverClose}
+            open={Boolean(anchorEl)}
+            disableRestoreFocus
+        >
+            {lineInfo === null && (
+                <Box height={2} minWidth={100}>
+                    <LinearProgress />
+                </Box>
+            )}
+
+            {lineInfo !== null && (
+                <Grid
+                    container
+                    rowSpacing={2}
+                    direction={'column'}
+                    alignItems={'center'}
+                >
+                    <>
+                        <Grid item>
+                            <Typography variant="h6">{lineId}</Typography>
+                        </Grid>
+
+                        <Grid item>
+                            <TableContainer component={Paper}>
+                                <Table size={'small'}>
+                                    <TableHead>
+                                        <TableRow>
+                                            <TableCell>
+                                                {intl.formatMessage({
+                                                    id: 'Intensity',
```

Comment:
We need :
Intensité (fr) = Current (en)

Correct key : "CURRENT"

Reply:
fixed